build/shade/_k-standard-goals.shade

```diff
@@ -154,13 +154,15 @@ default SAMPLES_PROJECT_GLOB = "samples/*/*.csproj"
 
       projectGlobs.AddRange(srcProjects);
     }
+
     if (!BuildSrcOnly && Directory.Exists("test"))
     {
-        projectGlobs.AddRange(Files.Include(TEST_PROJECT_GLOB).ToList());
+      projectGlobs.AddRange(Files.Include(TEST_PROJECT_GLOB).ToList());
     }
-    if (!BuildSrcOnly && Directory.Exists("samples") && Directory.EnumerateFiles("samples", "*.csproj", SearchOption.AllDirectories).Any())
+
+    if (!BuildSrcOnly && Directory.Exists("samples"))
```

Comment:
Put another way, alright if we require an empty samples folder to enable `SAMPLES_PROJECT_GLOB`. But requiring a (no-op) project is a bit much.